plugins/parsers/csv/parser.go

```diff
@@ -31,6 +31,10 @@ type Parser struct {
 	TimeFunc          func() time.Time
 }
 
+func (p *Parser) Name() string {
+	return "CSV parser - " + p.MetricName
```

Comment:
Would it be better to call this name function `Description`? I'd be interested in including the MetricName if it will help find the offending parse error.